plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -67,8 +67,7 @@ protected QueryRunner createQueryRunner()
             postgreSqlServer.close();
             postgreSqlServer = null;
         });
-        // TODO: https://github.com/trinodb/trino/issues/7031
-        return createPostgreSqlQueryRunner(postgreSqlServer, Map.of(), Map.of("topn-pushdown.enabled", "true"), REQUIRED_TPCH_TABLES);
```

Comment:
This change required using a `topNPushdownEnabled` session in tests in this class - which is the way it should be IMO.

Once it's enabled by default, we go back to using the default session.